### ISSUE TITLE
ERROR: Multiple errors in "Getting Started"/Wilson-Cowan tutorial

### ISSUE THREAD

**Issue by sbrincat** [Neuroblox/NeurobloxDocsHost#25]
In Cell 1:  
- `UndefVarError: `MetaDiGraph` not defined`.  Are we missing a `using MetaGraphs` line?  Adding that solved this.
- `MethodError: no method matching add_edge!(::MetaDiGraph{Int64, Float64}, ::Pair{WilsonCowan, WilsonCowan}; weight::Int64)`.  Solved by inserting this line from the old version (but we probably don't want to do that?): `create_adjacency_edges!(g, adj)`

In Cell 3:
- `A completed system is required. Call `complete` or `structural_simplify` on the system before creating an `ODEProblem``.  Solved by adding this line from the old version: `sys = structural_simplify(sys)`

In Cell 4:
- `UndefVarError: `state_timeseries` not defined`.  Could not figure out a solution for this one.

**Comment by harisorgn** [comment]
Thanks for the bug report Scott! All of these are already solved in Neuroblox v0.5, however we are experiencing some delay in getting this version online at JuliaHub. I'll let you know when it's up.

**Comment by harisorgn** [comment]
The new Neuroblox release (`v.0.5`) that is up-to-date with the tutorials on the docs page is online. Apparently you might need to re-add the JuliaHub registry for the new version to show up by doing  
``` julia
using Pkg
Pkg.add("PkgAuthentication")
using PkgAuthentication
PkgAuthentication.install("juliahub.com")
Pkg.Registry.add()
```
Please let me know if you are still experiencing these issues with the new release.
